DataAccess/Migrations/Migration_202407294_Add_PlacedBets.cs

```diff
@@ -0,0 +1,21 @@
+﻿using FluentMigrator;
+
+namespace DataAccess.Migrations
+{
+    [Migration(202407294)]
+    public class Migration_202407294_Add_PlacedBets : Migration
```

Comment:
let's name this in a meaningful way
e.g.: Migration_202407294_PlacedBets_AddBetPrice